archetypes/helidon/src/main/archetype/common/observability.xml

```diff
@@ -328,36 +322,36 @@ with the number of time it was triggered.
 
 ```
 curl http://localhost:8080/observe/metrics
-# HELP allRequests_total
+# HELP allRequests_totalb
```

Comment:
A typo with the extra "b" on the end?

Reply:
Yes that's a typo ! And unexpected change, thanks for catching that !